Makefile

```diff
@@ -65,15 +65,13 @@ coverage: ## check code coverage quickly with the default Python
 	coverage html
 	$(BROWSER) htmlcov/index.html
 
-docs: ## generate Sphinx HTML documentation, including API docs
-	rm -f docs/disruption_generator.rst
-	rm -f docs/modules.rst
-	sphinx-apidoc -o docs/ disruption_generator
```

Comment:
I don't think we are ready to have auto-generated API docs. They actually look like mess and there's not much of them anyway. We can easily re-activate this later once we actually have some docstrings.